src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -250,4 +268,13 @@ OptionRowLHN.propTypes = propTypes;
 OptionRowLHN.defaultProps = defaultProps;
 OptionRowLHN.displayName = 'OptionRowLHN';
 
-export default withLocalize(OptionRowLHN);
+export default compose(
+    withLocalize,
+    withReportCommentDrafts({
+        propName: 'comment',
+        transformValue: (drafts, props) => {
+            const draftKey = `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${props.reportID}`;
+            return lodashGet(drafts, draftKey, '');
+        },
+    }),
```

Comment:
I just test it working fine with `withOnyx`

```js
    withOnyx({
        comment: {
            key: props => `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${props.reportID}`,
        },
    }),
```

Reply:
@mollfpr It works with `withOnyx` but unit tests are failed.